.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["streetsidesoftware.code-spell-checker"]
```

Comment:
is this the project/extension for cspell? Or any reason we don't just use cspell here? I'd just like us to align on one project/extension if we can.

Reply:
Yes. It's the same one we recommend in `core-web-properties`. It's the official CSpell recommended extension.

Taking the same approach as the other repo where we aren't enforcing cspell on anyone currently, but it's a nice-to-have extension and config setup to we can keep our spelling mistakes to a minimum.